common/djangoapps/student/models.py

```diff
@@ -1589,3 +1589,11 @@ def user_can_skip_entrance_exam(cls, user, course_key):
             except EntranceExamConfiguration.DoesNotExist:
                 can_skip = False
         return can_skip
+
+
+class LanguageProficiency(models.Model):
+    """
+    Represents a user's language proficiency.
+    """
+    user_profile = models.ForeignKey(UserProfile, db_index=True, related_name='language_proficiencies')
```

Comment:
Thanks @ormsbee. Yes, allowing people to search across users and filter by language code is a use case we're planning to eventually support.